src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -570,8 +570,8 @@
   <data name="TupleElementLocationCountMismatch" xml:space="preserve">
     <value>If tuple element locations are specified, the number of locations must match the cardinality of the tuple.</value>
   </data>
-  <data name="TuplesNeedAtLeastTwoElements" xml:space="preserve">
-    <value>Tuples must have at least two elements.</value>
+  <data name="TuplesNeedAtLeastOneElement" xml:space="preserve">
+    <value>Tuples must have at least one element.</value>
```

Comment:
The language does not have a concept of "tuple type symbol", so it would not be appropriate to use that phrase in a language diagnostic.